platforms/mixin-common/build.gradle.kts

```diff
@@ -1,30 +1,31 @@
 plugins {
     id("dev.architectury.loom") version Versions.Mod.architecuryLoom
     id("architectury-plugin") version Versions.Mod.architecturyPlugin
-    id("io.github.juuxel.loom-quiltflower") version Versions.Mod.loomQuiltflower
+    id("io.github.juuxel.loom-vineflower") version Versions.Mod.loomVineflower
 }
 
 loom {
     accessWidenerPath.set(file("src/main/resources/terra.accesswidener"))
-    
+
     mixin {
         defaultRefmapName.set("terra.common.refmap.json")
     }
 }
 
 dependencies {
     shadedApi(project(":common:implementation:base"))
-    
+
     compileOnly("net.fabricmc:sponge-mixin:${Versions.Mod.mixin}")
     annotationProcessor("net.fabricmc:sponge-mixin:${Versions.Mod.mixin}")
     annotationProcessor("dev.architectury:architectury-loom:${Versions.Mod.architecuryLoom}")
-    
+
     minecraft("com.mojang:minecraft:${Versions.Mod.minecraft}")
     mappings("net.fabricmc:yarn:${Versions.Mod.yarn}:v2")
 }
 
 architectury {
-    common("fabric", "forge", "quilt")
+//    common("fabric", "forge", "quilt")
```

Comment:
Remove commented out code

Reply:
this is to make it easy to reenable quilt